docs/source/rules.rst

```diff
@@ -62,18 +63,52 @@ contain such values e.g.
             type: "eq"
             pattern : "customvalue"
 
+Managing Rules
+--------------
+
+To deploy a rule, use CLI ``st2 rule create ${PATH_TO_RULE}`` command, for example:
+
+.. code-block:: bash
+
+    st2 rule create /usr/share/doc/st2/examples/rules/sample_rule_with_webhook.yaml
+
+If the rule with the same name already exists, the above command will return an error:
+
+.. code-block:: bash
+    
+    ERROR: 409 Client Error: Conflict
+    MESSAGE: Tried to save duplicate unique keys (E11000 duplicate key error index: st2.rule_d_b.$name_1  dup key: { : "examples.webhook_file" })
+        
+To update the rule, edit the rule definition file and run ``st2 rule update`` command, as in the following example:
+    
+.. code-block:: bash
+    
+    st2 rule update examples.webhook_file /usr/share/doc/st2/examples/rules/sample_rule_with_webhook.yaml
+
+.. note::
+
+    **Hint:** It is a good practice to always edit the original rule file, so that keep your infrastructure in code. You still can get the rule definition from the system by ``st2 rule get <rule name> -j``, update it, and load it back.
 
-To deploy a rule, use CLI:
+To see all the rules, or to get an individual rule, use commands below:
 
 .. code-block:: bash
 
-    st2 rule create /opt/stackstorm/packs/examples/rules/sample_rule_with_webhook.yaml
-    st2 rule create /opt/stackstorm/examples/rules/sample_rule_with_webhook.yaml
     st2 rule list
     st2 rule get examples.webhook_file
 
+To undeploy a rule, run ``st2 rule delete ${RULE_NAME_OR_ID}``. For example, to undeploy the examples.webhook_file rule we deployed previously, run:
```

Comment:
Just a reminder for myself that I need to get #940 merged so user can also do `st2 rule disable <id>`.